pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -328,13 +332,21 @@ func (c *APIClient) connect() error {
 			informers.WithTweakListOptions(optionsForService),
 			informers.WithNamespace(common.GetResourcesNamespace()),
 		)
+		if err != nil {
```

Comment:
it should be ok

Reply:
`c.CertificateSecretInformerFactory, err = getInformerFactoryWithOption` instead of `c.CertificateSecretInformerFactory, _ = getInformerFactoryWithOption` (L331)

Reply:
Done at 218a3527fcef74ee6b1e72f587a1f0f7363e32dd!